Automattic/wp-calypso
# Add tests for the hasPlanWithAnalytics function

### Changes proposed in this Pull Request

A follow up of #45918. This PR adds tests for `hasPlanWithAnalytics` before refactoring its implementation in a next PR.

### Testing instructions

- Download the PR
- Review the code
- Run `yarn test-client:watch client/my-sites/site-settings/test/utils.js` and check all tests pass
